addons/account/tools/structured_reference.py

```diff
@@ -81,5 +119,6 @@ def is_valid_structured_reference(reference):
         is_valid_structured_reference_be(reference) or
         is_valid_structured_reference_fi(reference) or
         is_valid_structured_reference_no_se(reference) or
+        is_valid_structured_reference_nl(reference) or
```

Comment:
Not something about your code but i found sus that the sanitize_structured_reference has a special case only for belgian structure communication but maybe we are the only one to do weird thing with some +++/ :joy: